core/unit_test/TestAtomicViews.hpp

```diff
@@ -1044,17 +993,9 @@ bool LSEqualAtomicViewTest(const int64_t input_length) {
   T res = LSEqualAtomicView<T, DeviceType>(input_length, value, remainder);
   T resSerial = LSEqualAtomicViewCheck<T>(input_length, value, remainder);
 
-  bool passed = true;
-
-  if (resSerial != res) {
-    passed = false;
-
-    std::cout << "Loop<" << typeid(T).name()
-              << ">( test = RSEqualAtomicViewTest"
```

Comment:
Note the bug in the code being removed

Reply:
so we need iostream because gtest likes that?

Reply:
But I agree the rational for that change is much weaker than the other ones.